networking_generic_switch/exceptions.py

```diff
@@ -35,3 +35,7 @@ class GenericSwitchNetmikoNotSupported(GenericSwitchException):
 
 class GenericSwitchNetmikoConnectError(GenericSwitchException):
     message = _("Netmiko connection error: %(config)s, error: %(error)s")
+
+
+class GenericSwitchNetmikoConfigError(GenericSwitchException):
```

Comment:
Have you annotated code paths where this exception could now be thrown?

Reply:
No I haven't, I'll do that.